crates/common/src/lib.rs

```diff
@@ -3,6 +3,8 @@
 //!
 //! This includes many trivial wrappers around [Felt] which help by providing additional type safety.
 use anyhow::Context;
+#[cfg(any(feature = "test-utils", test))]
```

Comment:
Should we just remove these cfg entirely? I know they're meant to separate test / non-test stuff but maybe its fine with?

Reply:
We could, I'm curious if there's any significant impact on binary size :detective: